Makefile

```diff
@@ -0,0 +1,45 @@
+# See: https://gist.github.com/asukakenji/f15ba7e588ac42795f421b48b8aede63
+# For a list of valid GOOS and GOARCH values
+# Note: these can be overriden on the command line e.g. `make PLATFORM=<platform> ARCH=<arch>`
+PLATFORM=$(shell go env GOOS)
+ARCH=$(shell go env GOARCH)
+
+# build target, can be one of "production", "testing", "development"
+ENV=development
+WEBPACK_CONFIG=webpack/webpack.$(ENV).js
+
+.PHONY: help build-storybook build-client devops download-binaries tidy clean client-deps
+
+##@ Building
+
+build-storybook:
+	@yarn storybook:build
```

Comment:
Might be a good idea to remove some of the echo suppression.  i.e. take off `@` so we can see what it's doing and ensure there are no errors.  e.g. go mod tidy needed cd api in front.